lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -704,4 +717,51 @@ export default class BitcoinProcessor {
     // non sidetree transaction
     return sidetreeTxToAdd;
   }
+
+  /**
+   * Return transactions grouped by block height. EX: [[transactions in block1], [transactions in block2], [transactions in block3]]
+   * @param since Transaction number to query since
+   * @param numOfBlocks number of blocks worth of transactions to get at least
+   */
+  private async getAtLeastPageSizeBlocksWorthOfTransactionsSinceTransactionNumber (since: number | undefined): Promise<TransactionModel[][]> {
+    let beginTransactionNumber = since === undefined ? 0 : TransactionNumber.getBlockNumber(since);
```

Comment:
This line is initializing the ````beginTransactionBlock```` or ````beginTransactionTime```` right? If yes, then the starting block should be ````this.genesisBlockNumber````. Also, if yes, then the variable names for the corresponding end also needs to change.

Reply:
You are correct, idk why I named it transactionNumber. Good catch.